sdks/csharp/AgonesTestCore/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Agones;

```

Comment:
This is just a local manual test class. Consider replacing this with real (unit) tests.